pantheon-bundle/src/main/java/com/redhat/pantheon/model/document/Validations.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.pantheon.model.document;
+
+import com.redhat.pantheon.model.api.Child;
+import com.redhat.pantheon.model.workspace.WorkspaceChild;
+
+public interface Validations extends WorkspaceChild {
```

Comment:
I wonder if we could use the annontation on line 5 to define the primaryType for Validatiions node
@JcrPrimaryType("sling:OrderedFolder")